tests/pems/rsa_2048_pkcs1_sha256_cert_gen.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
```

Comment:
We don't want to just copy a different generate script that lives in the repo. Can you edit this script so that it _only_ produces the certificate that you created? Commenting out a bunch of lines that do other stuff is really messy.

Reply:
If this script is already generating the cert we need, could `S2N_RSA_2048_PKCS1_SHA256_CERT_CHAIN` potentially just path to the existing cert?

Reply:
The cert chain that gets generated from the original script does not work for the existing tests, so either we need a new script, or the tests need to be modified. Though I'm not 100% sure why the tests fail so I'll be investigating that. If only small change on tests are needed to make the tests pass, using existing cert chain could be better.

Reply:
Existing cert chain does work so I'll be using them instead. Just needed some modification to the tests